Error when running configure.py (CRLF error?)

When running the `python $DPCPP_HOME/llvm/buildbot/configure.py` script I got the following error:

```
-- Performing Test SUPPORTS_GNU_ZERO_VARIADIC_MACRO_ARGUMENTS_FLAG
-- Performing Test SUPPORTS_GNU_ZERO_VARIADIC_MACRO_ARGUMENTS_FLAG - Failed
-- Performing Test HAS_MAYBE_UNINITIALIZED
-- Performing Test HAS_MAYBE_UNINITIALIZED - Success
: not found/share/testing/sycl_workspace/llvm/llvm/cmake/config.guess: 6: 
: not found/share/testing/sycl_workspace/llvm/llvm/cmake/config.guess: 8: 
: not found/share/testing/sycl_workspace/llvm/llvm/cmake/config.guess: 28: 
: not found/share/testing/sycl_workspace/llvm/llvm/cmake/config.guess: 29: 
: not found/share/testing/sycl_workspace/llvm/llvm/cmake/config.guess: 40: 
: not found/share/testing/sycl_workspace/llvm/llvm/cmake/config.guess: 42: 
: not found/share/testing/sycl_workspace/llvm/llvm/cmake/config.guess: 54: 
: not found/share/testing/sycl_workspace/llvm/llvm/cmake/config.guess: 65: 
: not found/share/testing/sycl_workspace/llvm/llvm/cmake/config.guess: 68: 
/home/elias/share/testing/sycl_workspace/llvm/llvm/cmake/config.guess: 71: Syntax error: word unexpected (expecting "in")
CMake Error at cmake/modules/GetHostTriple.cmake:36 (message):
  Failed to execute
  /home/elias/share/testing/sycl_workspace/llvm/llvm/cmake/config.guess
Call Stack (most recent call first):
  cmake/config-ix.cmake:401 (get_host_triple)
  CMakeLists.txt:675 (include)


-- Configuring incomplete, errors occurred!
See also "/home/elias/share/testing/sycl_workspace/llvm/build/CMakeFiles/CMakeOutput.log".
See also "/home/elias/share/testing/sycl_workspace/llvm/build/CMakeFiles/CMakeError.log".
Traceback (most recent call last):
  File "/home/elias/share/testing/sycl_workspace/llvm/buildbot/configure.py", line 151, in <module>
    ret = main()
  File "/home/elias/share/testing/sycl_workspace/llvm/buildbot/configure.py", line 148, in main
    return do_configure(args)
  File "/home/elias/share/testing/sycl_workspace/llvm/buildbot/configure.py", line 112, in do_configure
    subprocess.check_call(cmake_cmd, cwd=abs_obj_dir)
  File "/usr/lib/python2.7/subprocess.py", line 190, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['cmake', '-G', 'Ninja', '-DCMAKE_BUILD_TYPE=Release', '-DLLVM_ENABLE_ASSERTIONS=ON', '-DLLVM_TARGETS_TO_BUILD=X86', '-DLLVM_EXTERNAL_PROJECTS=sycl;llvm-spirv;opencl-aot;xpti;libdevice', '-DLLVM_EXTERNAL_SYCL_SOURCE_DIR=/home/elias/share/testing/sycl_workspace/llvm/sycl', '-DLLVM_EXTERNAL_LLVM_SPIRV_SOURCE_DIR=/home/elias/share/testing/sycl_workspace/llvm/llvm-spirv', '-DLLVM_EXTERNAL_XPTI_SOURCE_DIR=/home/elias/share/testing/sycl_workspace/llvm/xpti', '-DLLVM_EXTERNAL_LIBDEVICE_SOURCE_DIR=/home/elias/share/testing/sycl_workspace/llvm/libdevice', '-DLLVM_ENABLE_PROJECTS=clang;llvm-spirv;sycl;opencl-aot;xpti;libdevice', '-DLIBCLC_TARGETS_TO_BUILD=', '-DSYCL_BUILD_PI_CUDA=OFF', '-DLLVM_BUILD_TOOLS=ON', '-DSYCL_ENABLE_WERROR=ON', '-DCMAKE_INSTALL_PREFIX=/home/elias/share/testing/sycl_workspace/llvm/build/install', '-DSYCL_INCLUDE_TESTS=ON', '-DLLVM_ENABLE_DOXYGEN=OFF', '-DLLVM_ENABLE_SPHINX=OFF', '-DBUILD_SHARED_LIBS=OFF', '-DSYCL_ENABLE_XPTI_TRACING=ON', '/home/elias/share/testing/sycl_workspace/llvm/llvm']' returned non-zero exit status 1
```

Which I had to overcome by running:

`dos2unix $DPCPP_HOME/llvm/llvm/cmake/config.guess`

--- a/sycl/doc/GetStartedGuide.md
+++ b/sycl/doc/GetStartedGuide.md
@@ -63,7 +63,7 @@ set DPCPP_HOME=%USERPROFILE%\sycl_workspace
 mkdir %DPCPP_HOME%
 cd %DPCPP_HOME%
 
-git clone https://github.com/intel/llvm -b sycl
+git clone --config core.autocrlf=false https://github.com/intel/llvm -b sycl
 ```
 
 ## Build DPC++ toolchain

